README.md

```diff
@@ -12,14 +12,10 @@ You will need something that implements the `BlockDevice` trait, which can read
 
 ```rust
 // Build an SD Card interface out of an SPI device
-let mut spi_dev = embedded_sdmmc::SdMmcSpi::new(sdmmc_spi, sdmmc_cs);
-// Try and initialise the SD card
-let block_dev = spi_dev.acquire()?;
-// The SD Card was initialised, and we have a `BlockSpi` object
-// representing the initialised card.
-write!(uart, "Card size is {} bytes", block_dev.card_size_bytes()?)?;
+let sdcard = embedded_sdmmc::SdCard::new(sdmmc_spi, sdmmc_cs);
```

Comment:
The delayer parameter is missing in this example.

Reply:
You can include README in the doc tests by adding this to lib.rs:
```
// Include README.md so its doc tests are run
#[doc = include_str!("../README.md")]
#[cfg(doctest)]
pub struct ReadmeDoctests;
```
But that triggers quite a lot of errors, because the examples in the README use some variables without initializing them. Of course, one could add those. But that would be a lot of boilerplate. In normal doc comments, that's not a problem, because this initialization code can be hidden from the documentation. That's not possible in the README.

Reply:
Copied the README example into an example file and made sure it actually compiles.